share/spice/coderwall/coderwall.js

```diff
@@ -58,8 +52,35 @@
             },
 
             templates: {
-                group: 'icon'
+                group: 'icon',
+                variants: {
+                    iconImage: 'large'
+                },
+                options: {
+                    chompContents: true,
```

Comment:
I've been playing around with these two helpers by passing `parseMD` as a [subexpression](http://handlebarsjs.com/expressions.html#subexpressions), but have been getting some strange behavior. This is because the custom helper returns unescaped inline tags (`<em>`,`<strong>`,`<code>`,`<a href>`). The built-in helpers are counting the markup as part of the max length. This results in unclosed tags with `{{{Condense}}}`  and inconsistent counting with `{{{Ellipsis}}}`, _i.e._, it's counting the url characters in `href`.

Any ideas on the best way to count only the visible characters?